README.md

```diff
@@ -1,44 +1,92 @@
+## Built With
 
-Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+This project is built with:
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+- ![React](https://img.shields.io/badge/-React-1d1919?style=flat&logo=react)
+- ![Redux](https://img.shields.io/badge/-Redux-1d1919?style=flat&logo=redux)
+- ![JavaScript](https://img.shields.io/badge/-JavaScript-1d1919?style=flat&logo=javascript)
+- ![HTML](https://img.shields.io/badge/-HTML-1d1919?style=flat&logo=html5)
+- ![CSS3](https://img.shields.io/badge/-CSS3-000000?style=flat&logo=css3&logoColor=ffffff&labelColor=1572B6)
 
-## Learn More
+## Getting Started
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+To get the content of this project locally you need to run these commands in your terminal.
+Before starting the steps below, be sure you have [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/) and [Node.js](https://nodejs.dev/learn/how-to-install-nodejs) installed in your computer.
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+1. `git clone https://github.com/Banstein/covid-19-tracker.git`
+2. `cd covid-19-tracker`
+3. `npm install` 
+  This will download all the dependancies of the project.
+4. `npm start` 
+  The page will open at http://localhost:3000 in your browser.
+
+
+It correctly bundles React in production mode and optimizes the build for the best performance. 
+
+## Test
+
+For tracking linter errors locally you need to follow these steps:
+
+- After cloning the project make sure you have run this command
+
+  > `npm install` command
+
+- For tracking the linter errors in CSS file run:
+
+  > `npx stylelint "\*_/_.{css,scss}"`
+
+- For tracking the linter errors in JavaScript file run:
+
+  > `npx eslint .`
+
+- And For running some unit tests:
+
+  > `npm test`
+
+## Authors
+
+👤 **Islam Bahnas**
+
+- GitHub: [@Banstein](https://github.com/Banstein)
+- Twitter: [@islam_bahnas](https://twitter.com/islam_bahnas)
+- LinkedIn: [islam-bahnas](www.linkedin.com/in/islam-bahnas)
+
+## :handshake: Contributing
+
+Feel free to check the [issues page](https://github.com/Banstein/covid-19-tracker/issues)
+
+## Show your support
+
+Give a :star: if you like this project!
+
+## Acknowledgments
+
+- Special thanks to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) for the design
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] The MIT link is broken, kindly fix this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.